catt/controllers.py

```diff
@@ -490,17 +494,18 @@ def skip(self):
 
 
 class PlaybackBaseMixin:
-    def play_media_url(self, video_url, **kwargs):
+    _cast: pychromecast.Chromecast
```

Comment:
Do we want to make these things class variables? That would break a whole slew of things.

Reply:
It's for mypy, and not intended as a class var. But I must admit that I've kind of lost control of this PR, as I struggle to understand how to get typing to work across python versions. As I see it, I would have to use old-style type hints, as I get syntax errors with annotations such as these (w. py3.4).

Reply:
You get syntax errors because of the variable annotations, which are new. Just keep the function annotations and we're good, we don't need to go full Python 3.6 on this.